expandFns.php

```diff
@@ -161,6 +161,17 @@ function title_capitalization($in, $caps_after_punctuation) {
     // ALL CAPS to Title Case
     $new_case = mb_convert_case($new_case, MB_CASE_TITLE, "UTF-8");
   }
+
+  // Implicit acronyms
+  $new_case = ' ' . $new_case . ' ';
+  $new_case = preg_replace_callback("~ [b-df-hj-np-tv-xz]{3,}(?= )~ui", 
```

Comment:
We could use `\s`, or possibly better still `\W` would match examples like the second half of `B/Gcvs`?
```suggestion
  $new_case = preg_replace_callback("~\W[b-df-hj-np-tv-xz]{3,}(?=\W)~ui", 
```